### ISSUE TITLE
When clicked or popover datetimepicker then HH:mm changed mm to 00

### ISSUE THREAD

**Issue by gottenwit** [nathanreyes/v-calendar#1057]
Hello @nathanreyes 

When selected time with minute then trigger minute reset to "00"

**Using** 
- VueJS3
- v-calendar/next

**Chrome Warning**
`The specified value "Invalid Date" does not conform to the required format.  The format is "HH:mm", "HH:mm:ss" or "HH:mm:ss.SSS" where HH is 00-23, mm is 00-59, ss is 00-59, and SSS is 000-999.`

**Scenario**
https://user-images.githubusercontent.com/52517915/151927999-0707e6e4-1d7a-4e6a-8db4-6f0820032252.mp4

**Code**
```
<v-date-picker
              class="inline-block h-full"
              v-model="state.exampleInfo[state.currentIndexSelected].exampledate"
              locale="th"
              :buddhist="buddhist"
              mode="dateTime"
              is24hr
              timezone="utc"
>
<template v-slot="{ inputValue, inputEvents }">
       <input class="bg-white border px-2 py-1 rounded" :value="inputValue" v-on="inputEvents" />
</template>
```

**Comment by Borna-f** [comment]
Up vote. I have a similar issue in the vue 3 too.

**Comment by Borna-f** [comment]
Update: seems like there is some issues with some yarn.lock materials. I had a working copy before so downgrading some stuff on yarn.lock solved the issue for me. 

But this is a workaround. Not sure if there exists an actual solution. Everything is in beta/alpha version in regard to vue 3 so not sure what is causing the issue and if there can be a solution. If so, it would be great.

With what I have I don't even need to import the style (import "v-calendar/dist/style.css").

**Comment by metadorm** [comment]
Same here, I've encountered this in alpha-7 and alpha-6. `mode="dateTime"` might be the cause.

**Comment by metadorm** [comment]
This problem is fixed for me in alpha-8.

**Comment by nathanreyes** [comment]
This should be fixed with `3.0.0-alpha.8`. I can re-open if needed.
